docs/interfaces/_src_methods_balances_transfer_.balancestransferargs.md

```diff
@@ -17,20 +19,20 @@
 
 ## Properties
 
-###  dest
+### dest
 
-• **dest**: *string*
+•  **dest**: string
 
-*Defined in [src/methods/balances/transfer.ts:13](https://github.com/paritytech/txwrapper/blob/6ef1ba4/src/methods/balances/transfer.ts#L13)*
+*Defined in [src/methods/balances/transfer.ts:13](https://github.com/paritytech/txwrapper/blob/5c4d9c5/src/methods/balances/transfer.ts#L13)*
 
 The recipient address, SS-58 encoded.
 
 ___
 
-###  value
+### value
 
-• **value**: *number | string*
+•  **value**: number \| string
```

Comment:
space?

Reply:
Same as above, hesitant to touch this since its auto generated and we didn't change the underlying file. Looking at the rich diff these changes seem to not affect how the markdown is displayed.